.github/workflows/pr-approval-check.yml

```
@@ -0,0 +1,129 @@
+name: PR Approval Check
+
+on:
+  pull_request_review:
+    types: [submitted]
+  pull_request:
```

```suggestion
  pull_request_target:
```

`pull_request_target` works just like `pull_request`, but runs a workflow from the base branch. If we used `pull_request`, anyone could change this workflow file in their PR.